wp-content/plugins/core/src/Blocks/Types/Content_Loop/Content_Loop.php

```diff
@@ -100,25 +103,25 @@ public function add_fields() {
 		// Setting Fields
 		//==========================================
 		$this->add_section( new Field_Section( self::SECTION_SETTINGS, __( 'Settings', 'tribe' ), 'accordion' ) )
-			->add_field(
-				new Field( self::NAME . '_' . self::LAYOUT, [
-					'type'            => 'image_select',
-					'name'            => self::LAYOUT,
-					'choices'         => [
-						self::LAYOUT_ROW     => __( 'Row', 'tribe' ),
-						self::LAYOUT_FEATURE => __( 'Feature', 'tribe' ),
-						self::LAYOUT_COLUMNS => __( 'Columns', 'tribe' ),
-					],
-					'default_value'   => self::LAYOUT_ROW,
-					'multiple'        => 0,
-					'image_path'      => sprintf(
-						'%sassets/img/admin/blocks/%s/',
-						trailingslashit( get_template_directory_uri() ),
-						self::NAME
-					),
-					'image_extension' => 'svg',
-				] )
-			);
+			 ->add_field(
```

Comment:
Weird green marks here, I checked the coded and it's not anything PHPStorm sees.. so ignore?